pinot-broker/src/main/java/com/linkedin/pinot/broker/requesthandler/BrokerRequestHandler.java

```diff
@@ -221,6 +225,15 @@ public BrokerResponse handleRequest(@Nonnull JSONObject request, RequesterIdenti
       return BrokerResponseFactory.getStaticNoTableHitBrokerResponse(ResponseType.BROKER_RESPONSE_TYPE_NATIVE);
     }
 
+    // Validate qps quota.
+    if (!_tableQueryQuotaManager.acquire(rawTableName)) {
```

Comment:
No need to log the pql. Just log the requestId. If possible, lets try to have the return at the end of the method. Avoids a bunch of duplicate code and makes it easy to read.